libvast/src/system/index.cpp

```diff
@@ -664,6 +665,7 @@ void index_state::send_report() {
   auto query_counters = get_query_counters(pending_queries);
   auto msg = report{
     .data = {
+      {"index.active_partitions_created", counters.active_partitions_created},
```

Comment:
I don't think this metric quite fulfills the needs I had when I created this internal feature request. Sorry for not making this clearer.

From a dashboarding perspective, I'd like to see metric emitted at the timepoint when a partition was rotated whose value is the number of events in the partition. The metadata should contain the schema name, so I can reliably observe whether partitions are rotated despite being underful, e.g., because the active partition timeout triggered.